### ISSUE TITLE
Include client-secret.json sample

### ISSUE THREAD

**Issue by scorchio** [qualman/gmail_delete_by_filter#1]
This project is super useful for me, thanks for putting it together - this is actually the first solution which I can use for cleaning up ~300k unneeded mails. 

For others it would be probably useful if you could include a sample of the client-secret.json file.

**Comment by larkoie** [comment]
Hi,
I can't make it work because of oauth credentials issues. I guess that my client-secret.json is not formated as expected by the script.
@scorchio could you please share a sample file based on your working case?

mine is like this : 
`{"installed":{"client_id":"123456789-nad12345879.apps.googleusercontent.com","project_id":"project-id-789456123","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"erererr45454545","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}`

Thanks

**Comment by scorchio** [comment]
@larkoie here you go:
```
{
  "installed": {
    "client_id": "<id>.apps.googleusercontent.com",
    "client_secret":"<redacted>",
    "redirect_uris": ["http://localhost", "urn:ietf:wg:oauth:2.0:oob"],
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://accounts.google.com/o/oauth2/token"
  }
}
```

**Comment by larkoie** [comment]
Thank you for this quick answer !
Unfortunately I still get the same error : 
```
Traceback (most recent call last):
  File "./deleter.py", line 139, in <module>
    main()
  File "./deleter.py", line 117, in main
    credentials = get_credentials()
  File "./deleter.py", line 51, in get_credentials
    store = oauth2client.file.Storage(credential_path)
AttributeError: 'module' object has no attribute 'file'
```
Does the script still work for you?

**Comment by larkoie** [comment]
OK the module file was not loaded, I had to add : 
`from oauth2client import file `

Looks good now

**Comment by qualman** [comment]
I implemented your example text in #4 . Thank you @scorchio ! #3 also solves the issue @larkoie was having :)
